covid-19-support/src/components/ResultsList.vue

```diff
@@ -6,38 +6,37 @@
         v-for="item in markers"
         v-bind:key="item.cartodb_id"
         class="resultItem"
-        :class="{ selected: item.cartodb_id == resource.resourceId, closedOne: item.isOpen == false }"
+        :class="{ selected: item.cartodb_id == resource.resourceId && showDetails, closedOne: item.isOpen == false }"
```

Comment:
The selected result is linked to the the selected red marker in the map so I think it makes sense to keep the result selected even if the details are hidden.
```suggestion
        :class="{ selected: item.cartodb_id == resource.resourceId, closedOne: item.isOpen == false }"
```